webroot/js/app/components/relation-view/relation-view.js

```diff
@@ -95,8 +203,13 @@ Vue.component('relation-view', {
             this.showRelationshipsPanel = true;
         },
 
+        /**
+         * helper function for template
+         *
+         * @return {Boolean} true if has at least a related object or a newly added object
+         */
         hasElementsToShow() {
-            return this.objects.length && this.addedRelations.length;
+            return this.objects.length || this.addedRelations.length;
```

Comment:
To avoid "surprises" (i.e. `TypeError` ghost):
```
 return (this.objects && this.objects.length) || (this.addedRelations && this.addedRelations.length);
```